src/main/java/com/ericsson/ei/exception/PropertyNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.ericsson.ei.exception;
+
+public class PropertyNotFoundException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    public PropertyNotFoundException(String string) {
```

Comment:
Sorry but the name of this String is invalid, name it message or something.

Reply:
Done